test/command/suite/select/filter/ordered_near_phrase/max_element_intervals/multi_token/all.expected

```diff
@@ -0,0 +1,62 @@
+table_create Entries TABLE_NO_KEY
+[[0,0.0,0.0],true]
+column_create Entries content COLUMN_SCALAR Text
+[[0,0.0,0.0],true]
+table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("unify_alphabet", false,                                   "unify_digit", false)'   --normalizer NormalizerNFKC121
+[[0,0.0,0.0],true]
+column_create Terms entries_content COLUMN_INDEX|WITH_POSITION Entries content
+[[0,0.0,0.0],true]
+load --table Entries
+[
+{"content": "abc1def1ghi"},
+{"content": "abc1def12ghi"},
+{"content": "abc1def123ghi"},
+{"content": "abc12def1ghi"},
+{"content": "abc12def12ghi"},
+{"content": "abc12def123ghi"},
+{"content": "abc1def123ghi"},
+{"content": "abc12def123ghi"},
+{"content": "abc123def123ghi"}
+]
+[[0,0.0,0.0],9]
+select Entries --filter 'content *ONP-1,0,5|5 "abc def ghi"' --output_columns '_score, content'
```

Comment:
Could you use different values for max element intervals to check suitable value is really use? For example, `5|6`?

Reply:
Changed to use `5|6` (and fix test cases)